petab/core.py

```diff
@@ -157,15 +157,10 @@ def get_optimization_to_simulation_parameter_mapping(self):
             self.parameter_df,
             self.sbml_model)
 
-
 def get_condition_df(condition_file_name):
-    """
-    Read the provided condition file into a `pandas.Dataframe`.
-    Conditions are rows, parameters are columns, conditionId is index.
-    """
+    """Read the provided condition file into a `pandas.Dataframe`"""
 
     condition_df = pd.read_csv(condition_file_name, sep='\t')
-    condition_df.set_index(['conditionId'])
```

Comment:
Why removed?

Reply:
we removed the set_index because the name of that column did not appear (as Yannik mentioned)